client/changelist/change.go

```diff
@@ -38,8 +38,10 @@ type TufChange struct {
 // TufRootData represents a modification of the keys associated
 // with a role that appears in the root.json
 type TufRootData struct {
-	Keys     data.KeyList `json:"keys"`
-	RoleName string       `json:"role"`
+	ReplaceKeys data.KeyList `json:"keys"`
```

Comment:
We probably want to keep the `keys` field for JSON unmarshalling for backwards compatibility, if someone has an old changefile lying around.  But maybe we can fix that in the unmarshaller.

I think we need to have `AddKeys` and `RemoveKeys` be separate though, because otherwise we can't add more keys or remove specific keys?